core/blockchain/blockchain.go

```diff
@@ -938,7 +938,7 @@ func (b *BlockChain) connectDagChain(node *blockNode, block *types.SerializedBlo
 		view := NewUtxoViewpoint()
 		view.SetViewpoints([]*hash.Hash{&node.hash})
 
-		stxos := make([]SpentTxOut, 0, countSpentOutputs(block))
+		stxos := []SpentTxOut{}
```

Comment:
does the modified version of ‘countspendoutputs’ method can not determinate size of slice? why do the removal?

Reply:
The size of this place has been changed to dynamic computing.